### ISSUE TITLE
关于small的复现问题

### ISSUE THREAD

**Issue by Joker-co** [Megvii-BaseDetection/YOLOX#724]
您好，我尝试了自己实现small的代码，有点奇怪的是，在前285个epoch的训练中，我的精度能够和官方精度对齐，但是在去掉mosaic, perspective和mixup，加上l1 loss的最后15个epoch，提点低了很多，最后结果只能达到39.2左右，请问最后15个epoch的训练有没有特殊的设置呢?

**Comment by Joker316701882** [comment]
@Joker-co What is your training setting? Can you provide your training command?

**Comment by Joker-co** [comment]
学习率的设置是：
lr: 0.00015625
momentum: 0.9
nesterov: True
weight_decay: 0.0005

总共训练300 epochs, 最后15个epoch不加mosaic，random_perspective和mixup

**Comment by Joker-co** [comment]
另外还想请问个事情，在yolox-tiny里self.input_scale = (416, 416)的作用是什么呢，貌似没有看到使用

**Comment by MSS-AH** [comment]
> 学习率的设置是： lr: 0.00015625 momentum: 0.9 nesterov: True weight_decay: 0.0005
> 
> 总共训练300 epochs, 最后15个epoch不加mosaic，random_perspective和mixup

你好，请问你解决了 L1loss 关闭的问题么？我最近也遇到了类似的问题，想关闭最后15个的L1 loss，但是在exp文件找不到相应的config的位置。

**Comment by FateScript** [comment]
You could find the comment [here](
https://github.com/Megvii-BaseDetection/YOLOX/blob/main/yolox/exp/yolox_base.py#L80-L81) @MSS-AH

**Comment by MSS-AH** [comment]
> You could find the comment [here](https://github.com/Megvii-BaseDetection/YOLOX/blob/main/yolox/exp/yolox_base.py#L80-L81) @MSS-AH

thanks a lot! didn't notice this line before 😂
